### ISSUE TITLE
insecure-skip-tls-verify doesnt work

### ISSUE THREAD

**Issue by evertimes** [MuhammedKalkan/OpenLens#122]
Hello, there is an option in kubeconfig `insecure-skip-tls-verify: true`,  that allows usage of self-signed certificates. Official Lens app supports this flag and my kubectl too, but when i try to access cluster using openlens, i've got error:

> Oops, something went wrong.
> Error: self signed certificate

**Comment by aridastidar** [comment]
+1 can confirm the behavior while using self-signed certificates to communicate over a proxy.
`E0216 16:16:40.385399 21944 proxy_server.go:147] Error while proxying request: x509: certificate signed by unknown authority x509: certificate signed by unknown authority`

OpenLens: 6.3.0
Extension API: 6.3.0
Electron: 19.1.9
Chrome: 102.0.5005.167
Node: 16.14.2

**Comment by aridastidar** [comment]
Found the solution in a different GH issue thread https://github.com/lensapp/lens/issues/5286#issuecomment-1275144084

TL;DR In the kubeconfig, remove `certificate-authority-data` section and add `insecure-skip-tls-verify: true`

**Comment by MuhammedKalkan** [comment]
not related binaries i guess
